Examples/options/geant_fullsim_fccee_pgun.py

```diff
@@ -14,7 +18,7 @@
 
 from Configurables import SimG4ConstantMagneticFieldTool
 field = SimG4ConstantMagneticFieldTool("SimG4ConstantMagneticFieldTool", FieldOn=True, 
-                                       IntegratorStepper="ClassicalRK4", FieldComponentZ=0.000,
+                                       IntegratorStepper="ClassicalRK4", FieldComponentZ=0.002,
```

Comment:
Could you add 
`from GaudiKernel.SystemOfUnits import tesla`
in the beginning and change this here to 
`FieldComponentZ=2*tesla`?